compiler/rustc_codegen_ssa/src/codegen_attrs.rs

```diff
@@ -463,6 +466,59 @@ fn codegen_fn_attrs(tcx: TyCtxt<'_>, did: LocalDefId) -> CodegenFnAttrs {
                     None
                 };
             }
+            sym::patchable_function_entry => {
+                codegen_fn_attrs.patchable_function_entry = attr.meta_item_list().and_then(|l| {
+                    let mut prefix = None;
+                    let mut entry = None;
+                    for item in l {
+                        let Some(meta_item) = item.meta_item() else {
+                            tcx.dcx().span_err(item.span(), "Expected name value pair.");
```

Comment:
For future reference, the message format doesn't use title-case nor periods. Maybe we should, but let's remain consistent with what we already have:

```suggestion
                            tcx.dcx().span_err(item.span(), "expected name value pair");
```